apps/aecore/lib/aecore/peers/sync.ex

```diff
@@ -39,6 +40,17 @@ defmodule Aecore.Peers.Sync do
     GenServer.call(__MODULE__, :add_valid_peer_blocks_to_chain)
   end
 
+  def add_unknown_peer_pool_txs(peers) do
+    Enum.each(peers, fn(peer) ->
+        deserialized_pool_txs =
+          peer
+          |> HttpClient.get_pool_txs()
+          |> elem(1)
+          |> Enum.map(fn(tx) -> Serialization.tx(tx, :deserialize) end)
```

Comment:
Deserialization should be handled by `HttpClient`.

Reply:
correct, thanks!